content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+title: darktable 3.2: containment effect!
+author: Nilvus
+slug: darktable-30
+date: 2020-08-10
+lede: 2013_Island_Kerlingarfjoll.jpg
+lede_author: AlicVB
+tags: announcement, darktable-release
+
+In other languages:
+[Français](https://linuxfr.org/news/darktable-3-2-l-effet-confinement),
+[Italiano](https://www.mauriziopaglia.it/darktable-3-2/),
+[Deutsch](https://www.bilddateien.de/blog/2020-08-10-darktable-3.2-auswirkungen-des-lockdowns.html).
+
+Not following the tradition, for the first time in its history, darktable sees its new major version 3.2, released a few months earlier, in the middle of summer. Containment of this year, already quite particular, has led to a record of contributions and improvements on darktable 3. On top of that, a major version, 3.4, is still scheduled for Christmas. 2020 will therefore be the first year in which the darktable team will have the pleasure to offer you two major versions.
+
+Before reading 3.2 features, for those who wish to refresh their memories about 3.0 features (and at the end of the article, 3.0.1 and 3.0.2 ones ; 3.0.2 is mainly bugfixes), you could read them on [3.0 article](https://www.darktable.org/2019/12/darktable-30/) released last Christmas.
+
+# A refined interface, down to the last details: for a professional look
+
+The interface has known a huge change with 3.0. darktable team has fine-tuned this work minutely. Thus, darktable 3.2 brings you an interface and new features that further enhance the user experience. The 3.0 style is confirmed and improved... for a more professional look. Several developers have worked hard to bring you a neat interface, starting with a complete rewriting of lighttable, which is also completed by nice performance improvements.
+
+Interface improvements are huge, some of them will be detailed below, such as the overhaul of the preferences, dynamic buttons or tooltips, and the biggest piece, the in-depth redesign of lighttable.
+
+This work has been an opportunity to completely restructure CSS, to make it clearer to read. Thus, it is now easier, with also much more comments, to find the part you want to personnalize on the interface. You will soon be able to grasp its usefulness even better when you will discover below the new preferences bring by darktable 3.2. In addition, this important work made it possible to go even further with the aim of making the interface as customizable as possible with CSS.
```

Comment:
Thanks for that one but I will keep much more. So i will add your others changes on my side.

Reply:
ok but "much more comments" really doesn't read well in English

Reply:
Generally if something can be counted, don't use 'much'.

Reply:
So all good, it can be counted (just by seeing CSS file).

Reply:
If it can be counted **don't** use "much" so you can say "many more comments" or "much more information" but not "much more comments" or "many more information"

Reply:
Oups, my bad